File: Famoser.MassPass.Data/Entities/Communications/Request/ContentEntityHistoryRequest.cs
using System;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class ContentEntityHistoryRequest : ApiRequest
    {
        public Guid ContentId { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/ContentEntityHistoryRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 23:01
 */

namespace Famoser\MassPass\Models\Request;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class ContentEntityHistoryRequest extends ApiRequest
{
    public $ContentId;
}